addon/components/bourbon-search-select-field.js

```diff
@@ -90,6 +89,15 @@ export default Component.extend(SelectMixin, {
     }
   }),
 
+  noResults: computed('searchList', function() {
+    console.log(this.get('searchList'))
```

Comment:
Remove `console.log` and observe `searchList.[]` so you can detect changes in the array

Reply:
lol...damn it.